CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## master
 
+* Added the ability to specify a custom directions client to be used by the CarPlayManager. ([#1834](https://github.com/mapbox/mapbox-navigation-ios/pull/1834/))
```

Comment:
This will end up being overwritten by #1836 anyways, but what the hey:

```suggestion
* Renamed `CarPlayManager(_:)` to `CarPlayManager(directions:eventsManager:)`, allowing you to pass in a custom `Directions` object to use when calculating routes. ([#1834](https://github.com/mapbox/mapbox-navigation-ios/pull/1834/))
```